src/instruments/src/EFB/TODCalculator/GroundSpeed/GroundSpeed.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * A32NX
+ * Copyright (C) 2020-2021 FlyByWire Simulations and its contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {connect} from 'react-redux';
+import Input from "../../Components/Form/Input/Input";
+import React from "react";
+import Card from "../../Components/Card/Card";
+import {TOD_CALCULATOR_REDUCER} from "../../Store";
+import {addTodGroundSpeed, removeTodGroundSpeed, setTodGroundSpeed} from "../../Store/action-creator/tod-calculator";
+
+import './GroundSpeed.scss'
+import Button, {BUTTON_TYPE} from "../../Components/Button/Button";
+
+const GroundSpeed = ({groundSpeed, addTodGroundSpeed, removeTodGroundSpeed, setTodGroundSpeed, ...props}) => {
```

Comment:
Make the props a type

Reply:
there is no props passed to this component from _outside_. Props are passed there from redux, so everything is within same file. props typing won't bring there any advantage I think

Reply:
Oh I thought it'd just clean up code, seeing that it is a redux thing, would it even be possible?